builder/docker/config.go

```diff
@@ -75,6 +75,8 @@ type Config struct {
 	// defaults to false if not set.
 	Privileged bool `mapstructure:"privileged" required:"false"`
 	Pty        bool
+	// Set the container runtime.
```

Comment:
Is there any additional documentation around why/when a user would want to use this option. I would say that this is an advance user option. Is that correct? 

If so maybe having a little documentation around what some valid options would look like, and a short explanation on why a user would want to set this configuration option would be helpful.

Reply:
In fact, we could consider this option as an advance user option for now.

The possible option (not exhaustive)
- [gVisor](https://gvisor.dev/docs/user_guide/quick_start/docker/) => `runsc`
- [sysbox](https://github.com/nestybox/sysbox) => `sysbox-runc`
- [kata-container](https://github.com/kata-containers/kata-containers) => `kata-runtime`

The [`docker run` reference](https://docs.docker.com/engine/reference/commandline/run/) doesn't tell more on that option.
> --runtime Runtime to use for this container

I will add following entries in the documentation for suggestions with the related website:

> Possible runtime are the following 
> 'runsc' for gVisor
> 'kata-container' for Kata-Container
> 'sysbox-runc' for Sysbox